app/views/tracks/concepts/index.html.haml

```diff
@@ -1,6 +1,16 @@
-#page-track-concepts
+#page-concepts-index
   = ViewComponents::Track::TopLevelNav.new(@track, :concepts)
 
-  .md-container
-    %h2 Concepts for #{@track.title}
+  .page-header
```

Comment:
(Well aware you're just quickly running through these and later fixing it, but for the logging, `header`)

Reply:
(I'd also expect everything but the nav to be inside `main` -- probably will come)